prediction_market_agent/agents/microchain_agent/call_api.py

```diff
@@ -0,0 +1,50 @@
+import requests
+
+class CallAPI:
```

Comment:
CallAPI needs to inherit from Function, [for example as here](https://github.com/gnosis/prediction-market-agent/blob/5304d2cb043072407627e5e5323436c45b74d052/prediction_market_agent/agents/microchain_agent/memory_functions.py#L13).